x-pack/platform/packages/shared/kbn-classic-stream-flyout/src/components/create_classic_stream_flyout/steps/name_and_confirm/name_stream_section.tsx

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiFormRow, EuiPanel, EuiSpacer, EuiTitle, EuiSelect, useEuiTheme } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { i18n } from '@kbn/i18n';
+import { css } from '@emotion/react';
+import type { ValidationErrorType } from '../../../../utils';
+import { StreamNameInput } from '../../../stream_name_input';
+import type { NameStreamSectionProps } from './types';
+
+const getValidationErrorMessage = (
+  validationError: ValidationErrorType,
+  conflictingIndexPattern?: string
+) => {
+  if (validationError === 'empty') {
+    return i18n.translate(
+      'xpack.createClassicStreamFlyout.nameAndConfirmStep.emptyValidationError',
+      {
+        defaultMessage:
+          'Please supply a valid text string for all wildcards within the selected index pattern.',
```

Comment:
Does one supply a valid string by entering it? If so, using `enter` seems like a more direct and actionable (less ambiguous) alternative.

```suggestion
          'Please enter a valid text string for all wildcards within the selected index pattern.',
```